src/schema/type/Annotation.graphql

```diff
@@ -0,0 +1,98 @@
+interface AnnotationTargetInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+}
+
+"An Annotation Target which refers to an external URL"
+type AnnotationURLTarget implements AnnotationTargetInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    url: URL!
+}
+
+"An Annotation Target which refers to a node in the CE"
+type AnnotationCETarget implements AnnotationTargetInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    "The item in the CE that this Annotation targets"
+    target: ThingInterface!
+    "The name of the field of the item in the CE which contains the target value"
+    field: String!
+    "In the case that the target is a fragment, the fragment value"
+    fragment: String
+}
+
+interface AnnotationMotivationInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+}
+
+# TODO: This will require creating an object of this type for each item in the enum AnnotationMotivation
+type AnnotationSchemaMotivation implements AnnotationMotivationInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    motivation: AnnotationMotivation!
+}
+
+"A custom motivation"
+type AnnotationCEMotivation implements AnnotationMotivationInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    # TODO: This should be rendered in jsonld as skos:Broader
+    parentMotivation: AnnotationMotivation!
+    motivation: String!
+}
+
+interface AnnotationBodyInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+}
+
+"http://www.w3.org/ns/oa#TextualBody"
+type AnnotationTextualBody implements AnnotationBodyInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    "http://www.w3.org/1999/02/22-rdf-syntax-ns#value"
+    value: String!
+    "http://purl.org/dc/elements/1.1/format"
+    format: String
+    "http://purl.org/dc/elements/1.1/language"
+    language: AvailableLanguage
+    # Should always be "http://www.w3.org/ns/oa#TextualBody"
+    "http://www.w3.org/1999/02/22-rdf-syntax-ns#type"
+    type: String
+    # TODO: Purpose (same as a motivation, can it be a custom value?)
```

Comment:
need feedback from @musicog if we want purpose

Reply:
Purpose is used to associated different bodies of an annotation with different motivations. Whether this is urgently required is a use-case question; I would be inclined to instead simplify a complex annotation with multi-purpose bodies into several simple annotations where each body's purpose aligns with the annotation motivation.

See also https://www.w3.org/TR/annotation-model/#motivation-and-purpose

Reply:
OK, for now we'll leave purpose out